public/locales/de/translation.json

```diff
@@ -0,0 +1,43 @@
+{
+  "intro": {
+    "text": [
+      "In diesem Spiel gibt es zwei Rollen: die investigierende und die lügende Person. Jede Runde wechselt sich diese Rolle.",
```

Comment:
Google Translate is suggesting `Ermittlungsperson`

Reply:
This makes no sense at all. Maybe `die ermittelnde Person` could work better.

Reply:
I'm not native, so you can choose :)

Reply:
I will change every instance of `investigierende` to `ermittelnde` which is a more fitting word.